app/uk/gov/hmrc/bankaccountverificationexamplefrontend/example/PetDetails.scala.html

```diff
@@ -98,6 +110,57 @@ <h1 class="govuk-heading-xl">@messages("petDetailsPage.heading")</h1>
             name = detailsForm("description").name, id = detailsForm("description").name,
             errorMessage = detailsForm("description").error.map(fe ⇒ ErrorMessage(content = HtmlContent(messages(fe.message))))))
 
+        <br/>
+        <br/>
```

Comment:
`<br>` for line breaks instead of closing the `<p>` and then changing to an `<h2>` look nasty from an HTML point of view.

Reply:
Removed the <br /> tags.